src/Sylius/Bundle/AdminBundle/Resources/views/PaymentMethod/_form.html.twig

```diff
@@ -6,7 +6,26 @@
             <div class="two fields">
                 {{ form_row(form.code) }}
                 {{ form_row(form.gateway) }}
+
+                {% set currentGateway='' %}
+                {% for  gateway in form.vars.gateways %}
+                    {% set gatewayHtml= form_widget(attribute(form,gateway)) %}
+
+                    {% if gateway== form.gateway.vars.value %}
+                        {% set currentGateway=gatewayHtml %}
+                    {% endif %}
+
+                    <input type="hidden" data-form-gateway="{{ gateway }}" value="{{ gatewayHtml|e }}"/>
+                {% endfor %}
+
+                <div data-form-type="gateway">
+                    {% if  currentGateway is not empty %}
+                        {{ currentGateway|raw }}
+                    {% endif %}
```

Comment:
All this code should have cs fixes applied:
- each ``=``, ``==`` should be surrounded by spaces
- ``if``, ``for``, ``set`` etc should be followed by **one** space